src/Observable.ts

```diff
@@ -37,6 +37,10 @@ import {ErrorObservable} from './observable/throw';
 import {AjaxCreationMethod} from './observable/dom/ajax';
 import {WebSocketSubject} from './observable/dom/webSocket';
 
+export type ObservableOrPromise<T> = Observable<T> | Promise<T>;
```

Comment:
Will there be cases of this type import would be required, but importing `Observable` is not required? I was thought of having this as separate module other than `Observable` if there's those kind of cases. In case of operator it won't by all operator would need to import `Observable`.